docs/integrations/ctest.md

```diff
@@ -0,0 +1,39 @@
+# CTest
+
+## Recording test results
+
+CTest does not natively produce JUnit compatible test report files, so you should use an extra tool such as [`xsltproc`](http://xmlsoft.org/xslt/xsltproc.html) and [`ctest-to-junit`](https://github.com/rpavlik/jenkins-ctest-plugin/blob/master/ctest-to-junit.xsl) to convert them for use with Launchable.
```

Comment:
As we all know, JUnit report files produced must contain key bits of information in the right way in order for the subset to work. Yet I notice that the documentation is somewhat open-ended about how to produce JUnit report files. Isn't that inviting a trouble, or did I miss something that makes this process actually more robust than I thought?

* A quick look at the referenced XSLT stylesheet shows test names are having their `-` replaecd to `_`. If we are matching with test names, that creates a problem, I'd think.
* Given the error prone nature of this process, I wouldn't use a phrase like "such as", which implies other tools are OK. Or perhaps be more specific about what this format needs to satisfy.